doc/locale/ja/LC_MESSAGES/developer-guide/api/functions/TSHttpOverridableConfig.en.po

```diff
@@ -425,11 +425,6 @@ msgstr ":ts:cv:`proxy.config.http.cache.when_to_revalidate`"
 msgid ":ts:cv:`proxy.config.http.cache.open_write_fail_action`"
 msgstr ":ts:cv:`proxy.config.http.cache.when_to_revalidate`"
 
-#: ../../../developer-guide/api/functions/TSHttpOverridableConfig.en.rst:150
-#, fuzzy
-msgid ":ts:cv:`proxy.config.http.redirection_enabled`"
-msgstr ":ts:cv:`proxy.config.http.flow_control.enabled`"
```

Comment:
Actually, nm, that was a typo in the original docs file, those lines should both be removed.